### PR TITLE
LC-2050 Implement user invite function-8

### PR DESCRIPTION
**Pull request by neeraj-jain-cbo**

### CODE REVIEW

File: src/main/resources/templates/signup/requestInvite.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
             <a href="/login" class="link-back">Back</a>
             <div th:if="${status}" class="error-summary" role="alert"  aria-labelledby="error-summary-heading-example-1" tabindex="-1">
                 <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-2">
-                    There was a problem inviting this email address
```
Comment (jaylco) [inline src/main/resources/templates/signup/requestInvite.html:10]: 'was' makes more sense here

File: src/main/resources/templates/signup/requestInvite.html [hunk 2]
```diff
@@ -16,7 +16,7 @@ <h2 class="heading-medium error-summary-heading" id="error-summary-heading-examp
             <div th:if="${#fields.hasErrors('${requestInviteForm.*}')}" class="error-summary" role="alert"
                  aria-labelledby="error-summary-heading-example-1" tabindex="-1">
                 <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
-                    There was a problem with your email address
+                    There is a problem with your email address
```
Comment (jaylco) [inline src/main/resources/templates/signup/requestInvite.html:19]: 'was' makes more sense here

File: src/main/java/uk/gov/cabinetoffice/csl/util/ApplicationConstants.java [hunk 3]
```diff
@@ -4,21 +4,21 @@ public class ApplicationConstants {
     
     public static final String STATUS_ATTRIBUTE = "status";
 
-    public static final String ENTER_TOKEN_ERROR_MESSAGE = "Incorrect organisation or token";
+    public static final String ENTER_TOKEN_ERROR_MESSAGE = "Incorrect organisation or token.";
 
-    public static final String NO_SPACES_AVAILABLE_ERROR_MESSAGE = "No spaces available for this token. Please contact your line manager";
+    public static final String NO_SPACES_AVAILABLE_ERROR_MESSAGE = "No spaces available for this token. Please contact your line manager.";
 
-    public static final String CHANGE_EMAIL_ERROR_MESSAGE = "There was a problem with changing your email, please try again later";
```
Comment (jaylco) [inline src/main/java/uk/gov/cabinetoffice/csl/util/ApplicationConstants.java:11]: 'was' makes more sense here

Comment (neeraj-jain-cbo) [reply src/main/java/uk/gov/cabinetoffice/csl/util/ApplicationConstants.java:11]: updated

**Review by jaylco — APPROVED** [review]

**Pull request merged**